_posts/2016-04-19-JabRef3.3.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Release of JabRef 3.3
+id: JabRef3.3
+author: "[JabRef Developers](https://github.com/orgs/JabRef/teams/developers)"
+---
+
+We are pleased to announce the release of JabRef version 3.3!
+
+Since yesterday, it can be downloaded from [FossHub](http://www.fosshub.com/JabRef.html), the official JabRef download site.
+
+This release contains new and improved features such as:
+- file-based BibTeX and BibLaTeX modes 
+- improved import from ArXiv and Ovid
+- better handling of PDF files
+- better integration of OpenOffice/LibreOffice style files
+- faster search
+- easier installation under windows
+- translation into Swedish
+- online help
+- improved support on Mac OS X
+
+It also includes about 40 bug fixes and the removal of a few features (mostly outdated).
+Details are listed on [gitHub](https://github.com/JabRef/jabref/blob/master/CHANGELOG.md).
```

Comment:
I'd link to https://github.com/JabRef/jabref/blob/v3.3/CHANGELOG.md - which is the fixed changelog for this release.